src/lms-harmonizer/tests_integration_pgsql/pgsql_orchestrator.py

```diff
@@ -79,7 +81,7 @@ def _psql_parameters_from(config: PgsqlServerConfig) -> List[str]:
 def _execute_sql_against_master(config: PgsqlServerConfig, sql: str):
     _run(
         config,
-        [config.psql_cli, *_psql_parameters_from(config), "-d", "postgres", "-c", f"'{sql}'"],
```

Comment:
psql didn't like this as all sql commands were being received with the single quotes included (!)

Reply:
I had actually added that to fix a bug - it was failing for me without that. Might be something to do with my docker script then. This change may force me to update that docker script.